lib/debugger/debugger.dart

```diff
@@ -34,18 +34,14 @@ import '../ui/primer.dart';
 class DebuggerScreen extends Screen {
   DebuggerScreen()
       : super(name: 'Debugger', id: 'debugger', iconClass: 'octicon-bug') {
+    debuggerState = new DebuggerState();
```

Comment:
nit: make debuggerState final now that it can be.

Reply:
done